### PR TITLE
fix:[CORE-1635] Notification for spectro cloud aws collector

### PR DESCRIPTION
**Pull request by AishwaryaKulkarni97**
# Description

Send notification when role is not given to collector for AWS

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

1. Remove sts role from IAM Roles. Remove highlighted section in screenshot
image

2. Run AWS collector from local

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

File: jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java [hunk 1]
```diff
@@ -157,7 +157,7 @@ public static void omitOpsAlert() {
             Map<String, List<String>> assetPermissionMapping = new HashMap<>();
             for (ErrorVH errorVH : entry.getValue()) {
                 List<String> permissionIssues = assetPermissionMapping.get(errorVH.getType());
-                if (errorVH.getType().equals("phd") && (errorVH.getException().contains("AccessDeniedException") || errorVH.getException().contains("SubscriptionRequiredException"))) {
+                if ((errorVH.getType().equals("phd") && (errorVH.getException().contains("AccessDeniedException") || errorVH.getException().contains("SubscriptionRequiredException")))||(errorVH.getType().equals("all") && errorVH.getException().contains("AWSSecurityTokenService"))) {
```
Comment (plyubich) [inline jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java:160]: @AishwaryaKulkarni97 please properly format spacings in the expression.

**Review by plyubich — DISMISSED** [review]
Please format your code before commit.

File: jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java (hunk 1 above)
Comment (AishwaryaKulkarni97) [reply jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java]: Addressed in commit

**Review by santhosh-challa — APPROVED** [review]
LGTM

File: jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java [hunk 2]
```diff
@@ -40,113 +38,117 @@
  */
 public class ErrorManageUtil {
 
-    /** The log. */
+    /**
+     * The log.
+     */
```
Comment (mkinchin-paladin) [inline jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java:43]: Thanks for applying basic style.
Let's get rid of these useless comments - waste of visual space and IDE and distraction.

Comment (mkinchin-paladin) [reply jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/file/ErrorManageUtil.java:43]: Same for all similar basic comments below.

**Review by mkinchin-paladin — CHANGES_REQUESTED** [review]
How it was tested? Please add test results

**Comment by mkinchin-paladin** [comment]
@AishwaryaKulkarni97 do you have any updates for this PR?

**Comment by AishwaryaKulkarni97** [comment]
> @AishwaryaKulkarni97 do you have any updates for this PR?

Updated

**Pull request merged**